pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/DistinctCountRawThetaSketchAggregationFunction.java

```diff
@@ -53,7 +53,8 @@ public String extractFinalResult(List<ThetaSketchAccumulator> accumulators) {
     int numAccumulators = accumulators.size();
     List<Sketch> mergedSketches = new ArrayList<>(numAccumulators);
 
-    for (ThetaSketchAccumulator accumulator : accumulators) {
+    for (Object object : accumulators) {
```

Comment:
Can we please test this if this indeeds helps? 

As discussed offline, as a result of the aggregationFunction.merge() step, all Sketch objects should be converted to ThetaSketchAccumulator. So can you clarify why we saw this?

Reply:
Good question! It could be possible that there is only 1 old server returning a non-empty intermediate result, which skipped the merge step in broker.

I've tested this code by setting up a local cluster and running 2 servers with old version and 1 broker with new version, and I don't see exception any more. So we should be good with this hotfix PR.